### ISSUE TITLE
Decimals for VEGA token amounts

### ISSUE THREAD

**Issue by barnabee** [vegaprotocol/token-frontend#885]
General comment where front end shows VEGA token amounts specifically. 

Unlike generic ERC20s, we know the rough range of value for VEGA and never need to show all 18dps. 4 decimal places is precision to less than $0.01, for example. Two or three places is probably fine even, in the UI. 

Some pages (staking for example) are quite hard to read because all the decimals wrap. If people want exact values to 18dps they can use the API, or we could show the detail on hover if we’re feeling fancy. 

Either way we should avoid wrapping numbers across multiple lines as a result of showing decimals to fractions of a cent of value.

**Comment by campbellssource** [comment]
A bunch have already been set to 2DP. But not deployed to testnet.
But I really like using the principle of where we know we are talking about VEGA (and assuming a VEGA is not worth $60k) we can cap it.
One exception might be were list lots of assets together, e.g. all your collateral balances. we might show all DP until we have a better source of where the handle is (e.g. how many DP are relevant)

**Comment by barnabee** [comment]
Yeah makes total sense - hard coding for VEGA in an generic field that could be any ERC20 in Console is perhaps a bit sloppy and makes sense not to do that until we have something that works for other assets too but where we know we’re always talking about VEGA we have a nice shortcut.

**Comment by campbellssource** [comment]
Closing this now as most mentions of VEGA on token site now show a 2 DP, there are some exceptions as mentioned above.
Where the value is shown in a list with other asset balances and we don't know the appropriate level to truncate at.
Where we think the user might want to see the full number (withdrawing balances, seeing rewards).
I've also made a note in Design file about the rule. 
I suggest we look into getting souring information about where to truncate other numbers later (perhaps at the same time as an oracle for XXX to home currency)
